dcpy/connectors/edm/publishing.py

```diff
@@ -283,11 +283,15 @@ def validate_or_patch_version(
 
     if version_already_published:
         if is_patch:
-            return versions.bump(
+            patched_version = versions.bump(
                 previous_version=version,
```

Comment:
This is the culprit of the bug: this line takes the input version, ex: `24Q1`, and patches to `24Q1` even though it may exist.

Reply:
A little unclear to me - how does this fix it? It seems to offer validation, but I don't see how this actually changes behavior

Reply:
Oops